rosbag2_transport/test/rosbag2_transport/mock_sequential_writer.hpp

```diff
@@ -55,6 +55,15 @@ class MockSequentialWriter : public rosbag2_cpp::writer_interfaces::BaseWriterIn
       snapshot_buffer_.push_back(message);
     }
     messages_per_topic_[message->topic_name] += 1;
+    messages_per_file_ += 1;
+    if (messages_per_file_ == 5) {  // "Split" the bag every 5 messages
```

Comment:
I would recommend to define some constant aka `constexpr size_t MAX_MESSAGES_PER_FILE = 5` somewhere in `mock_sequential_writer` and `mock_sequantial_reader` as public member and refer on it everywhere in tests.

Reply:
Done in 9e2cd6b.

Reply:
I meant to define getters for maximum messages per file aka
```cpp
size_t max_messages_per_file();
```
and use it in `TEST_F(RosBag2PlayTestFixture, read_split_callback_is_called)` and `TEST_F(RecordIntegrationTestFixture, write_split_callback_is_called)` as well.

Reply:
I added getters for the constants, but the constant value is not used by the tests so there's nowhere to use them.